docs/vsphere-golden-image.md

```diff
@@ -101,26 +102,20 @@ In case no firewall rule exist, you must create it by running the following Powe
     New-NetFirewallRule -DisplayName 'OpenSSH Server (sshd)' -LocalPort 22 -Enabled True -Direction Inbound -Protocol TCP -Action Allow 
 ```
 
-### Add the Administrator Public Key
+### Set up key-based authentication
```

Comment:
I suggest removing this section. It is not really adding any useful information here.